atlasdb-config/src/test/java/com/palantir/atlasdb/config/ShouldRunBackgroundSweepSupplierTest.java

```diff
@@ -107,7 +107,7 @@ public void liveReload() {
     private static boolean runBackgroundSweep(
             TargetedSweepInstallConfig tsInstall, TargetedSweepRuntimeConfig tsRuntime, SweepConfig bgSweepConfig) {
         return new ShouldRunBackgroundSweepSupplier(
-                        () -> createRuntimeConfig(tsRuntime, bgSweepConfig).sweep(), tsInstall.enableSweepQueueWrites())
```

Comment:
This is correct (and you have to do this), but some of the other tests will break :(